webhook_features/src/features/summary_comment.rs

```diff
@@ -70,7 +70,7 @@ impl Feature for SummaryCommentFeature {
                 let pr_number = payload["number"]
                     .as_u64()
                     .ok_or(DrahtBotError::KeyNotFound)?;
-                refresh_summary_comment(ctx, repo, pr_number).await?
+                refresh_summary_comment(ctx, repo, pr_number, action == "synchronize").await?
```

Comment:
So if there are two pushes in 10 minutes, there will be several requests?

Reply:
Yes, but it won't do anything the second time. You can only request a review from someone if they haven't left one yet or if they've left a review after a review request (from what I've observed).